src/Modules/OrchardCore.Commerce/Drivers/ProductAttributeFieldDriver.cs

```diff
@@ -23,7 +23,6 @@ public abstract class ProductAttributeFieldDriver<TField, TFieldSettings> : Cont
             GetEditorShapeType(context), model =>
             {
                 var settings = new TFieldSettings();
-                context.PartFieldDefinition.PopulateSettings(settings);
```

Comment:
Why is this not needed anymore?

Reply:
`PopulateSettings` has been removed from OC. In this case it was also pointless, because `EditProductAttributeFieldViewModel` doesn't have any descendants and all of its properties are manually set in the lines below.

Reply:
OK then.